src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/JudicialUsersFunctionalTest.java

```diff
@@ -1,60 +1,110 @@
 import uk.gov.hmcts.reform.judicialapi.util.CustomSerenityRunner;
 import uk.gov.hmcts.reform.judicialapi.util.FeatureConditionEvaluation;
 import uk.gov.hmcts.reform.judicialapi.util.ToggleEnable;
 
+import javax.sql.DataSource;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
 
+import static java.lang.System.getenv;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.springframework.http.HttpStatus.FORBIDDEN;
 import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
 
 @RunWith(CustomSerenityRunner.class)
 @WithTags({@WithTag("testType:Functional")})
 @Slf4j
 public class JudicialUsersFunctionalTest extends AuthorizationFunctionalTest {
 
-    private static String PATH = "/refdata/judicial/users/fetch?page_size=" + 10 + "&page_number=" + 0;
+    private DataSource dataSource;
+    private JdbcTemplate template;
+
+    private static final String INSERT_MULTIPLE_USERS =
+              "INSERT INTO judicial_user_profile (per_id, personal_code, appointment, known_as, surname, full_name, "
+            + "ejudiciary_email, extracted_date, sidam_id) "
+            + "VALUES (12344, 'Emp1', 'Magistrate', 'Test', 'Test1','Test Test1', 'abc@gmail.com', current_timestamp, "
+            + "'44862987-4b00-e2e7-4ff8-281b87f16bf9') ON CONFLICT (per_id) DO NOTHING;";
+
+    private static final String DELETE_TEST_USERS = "DELETE FROM judicial_user_profile WHERE "
+            + "sidam_id IN ('44862987-4b00-e2e7-4ff8-281b87f16bf9');";
+
     public static final String FETCH_USERS = "JrdUsersController.fetchUsers";
 
-    //TODO: add successful response scenario
+    public void cleanUp() {
+        template.update(DELETE_TEST_USERS);
+        try {
+            template.getDataSource().getConnection().close();
+        } catch (Exception e) {
+            log.info("DB connection not found");
+        }
+    }
+
+    private void dbSetup() {
+        String host = getenv("DATABASE_HOST");
+        String port = getenv("DATABASE_PORT");
+        String dbName = getenv("DATABASE_NAME");
+        String username = getenv("DATABASE_USER");
+        String password = getenv("DATABASE_PASS");
+        dataSource = DataSourceBuilder.create()
+                .driverClassName("org.postgresql.Driver")
+                .url("jdbc:postgresql://" + host + ":" + port + "/" + dbName)
+                .username(username)
+                .password(password)
+                .build();
+
+        template = new JdbcTemplate(dataSource);
+        template.update(INSERT_MULTIPLE_USERS);
+    }
 
     @Test
     @ToggleEnable(mapKey = FETCH_USERS, withFeature = true)
-    public void shouldReturnDataNotFound() {
-        Response fetchResponse = judicialApiClient.getMultipleAuthHeadersInternal(ROLE_JRD_SYSTEM_USER)
-                .body(getUserRequest())
-                .post(PATH)
-                .andReturn();
+    public void shouldReturn200() {
```

Comment:
It is a good practice to have a descriptive name for the test method. Alternatively, you could use the `@DisplayName` to explain the scenario being tested in the test case. `shouldReturn200` only indicates the outcome but it doesn't tell which scenario is being tested.

Reply:
addressed here dc7b300